cmd/run.go

```diff
@@ -143,84 +146,51 @@ func Run(isDebug *bool) *cli.Command {
 			}()
 
 			logger := makeLogger(*isDebug)
-			inputPath := c.Args().Get(0)
-			if inputPath == "" {
-				errorPrinter.Printf("Please give a task or pipeline path: bruin run <path to the task definition>)\n")
-				return cli.Exit("", 1)
+			// Initialize runConfig with values from cli.Context
+			runConfig := &scheduler.RunConfig{
+				Downstream:   c.Bool("downstream"),
+				StartDate:    c.String("start-date"),
+				EndDate:      c.String("end-date"),
+				Workers:      c.Int("workers"),
+				Environment:  c.String("environment"),
+				Force:        c.Bool("force"),
+				PushMetadata: c.Bool("push-metadata"),
+				NoLogFile:    c.Bool("no-log-file"),
+				FullRefresh:  c.Bool("full-refresh"),
+				UseUV:        c.Bool("use-uv"),
+				Tag:          c.String("tag"),
+				ExcludeTag:   c.String("exclude-tag"),
+				Only:         c.StringSlice("only"),
 			}
 
-			startDate, err := date.ParseTime(c.String("start-date"))
-			logger.Debug("given start date: ", startDate)
+			var startDate, endDate time.Time
+
+			var err error
+			startDate, endDate, inputPath, err := Validation(runConfig, c.Args().Get(0), logger)
```

Comment:
I'd give this function a more explicit name, sth like `ValidateRunConfig` or even put the function in the `RunConfig` struct maybe?